dask/dataframe/core.py

```diff
@@ -1413,11 +1413,17 @@ def repartition(
         The second-to-last item is the lower bound of the last partition, and the last
         (extra) item is the upper bound of the last partition.
 
+        Also note that `partition_size` will not necessarily be reflected on disk.
```

Comment:
Can you put this under the `partition_size` parameter description?

I'd phrase it as something like: "Note that the size reflects the number of bytes used as computed by :ref:`pandas.DataFrame.memory_usage`, which will not necessarily match the size on disk."

Reply:
I went ahead and moved the clarification. I copied the same formatting, but the reference to `pandas.DataFrame.memory_usage` doesn't seem to work properly.

Reply:
Indeed, I don't think the `:ref:` syntax works directly with pandas docs. We can consider adding it in the "See Also" section to link to pandas docs. I'll add suggestions. :)